client/src/pages/LoginPage/LoginPage.js

```diff
@@ -9,16 +9,27 @@ import { LoginDiv } from "./LoginPage.style.js";
 
 class LoginPage extends Component {
   render() {
+    console.log(this.props);
     return (
       <React.Fragment>
         <Header />
         <LoginDiv>
-          <p>Login</p>
-          <InputBox type="text" name="Username" />
-          <InputBox type="password" name="Password" />
-          <Link to="/predictions">
+          <form onSubmit={this.handleSubmit}>
+            <InputBox
+              type="text"
+              value={this.props.data.username}
+              defaultValue="ben"
+              onChange={this.props.functions}
+            />
+            <InputBox
+              type="password"
+              value={this.props.data.password}
```

Comment:
The only concern is setting the value of the input field, so as we discussed, @BenNewman90 is handling this issue in the next branch.